### DISCUSSION TITLE
Suggestion: Portuguese names

### DISCUSSION THREAD

**Discussion by JSlvr** [voxeltycoon/issues#470] [UI&UX] [answered]
Hi,

I've created a list with portuguese city names, names and surnames that I would love to see being added to the game :)

[pt.zip](https://github.com/voxeltycoon/suggestions/files/6344614/pt.zip)

Here are some pictures of it working:
![Portuguese_names](https://user-images.githubusercontent.com/82893262/115421647-08ffb500-a1f4-11eb-9fbe-1d2304896d90.png)
![Portuguese_cities_1](https://user-images.githubusercontent.com/82893262/115421652-09984b80-a1f4-11eb-9d59-7b35fc37cb6f.png)
![Portuguese_cities_2](https://user-images.githubusercontent.com/82893262/115421640-07ce8800-a1f4-11eb-9900-2edd86ceb947.png)
![Portuguese_cities_3](https://user-images.githubusercontent.com/82893262/115421645-08671e80-a1f4-11eb-9394-ae2684e2fe98.png)
![Portuguese_names_1](https://user-images.githubusercontent.com/82893262/115421651-08ffb500-a1f4-11eb-8173-55c17696c087.png)

Thank you so much for your time :)
Pedro Assis

**Comment by victorleblais** [comment]
Thanks a lot for taking the time to do that !

**Comment by JSlvr** [comment]
I would like to add that as I played today, with the modification, I encountered several names (with a yellow background) that are not in the list that I created. I don't understand where they are coming from...but I didn't actually had the time to troubleshoot it.

**Comment by victorleblais** [comment]
Names with yellow background are the names of supporters that made a donation and/or support via Patreon. You can't change or remove them.

**Comment by JSlvr** [comment]
ooooh, ok ok. I tought it was an error in my list or something. Thanks for the clarification :)

**Comment by victorleblais** [comment, marked as answer]
The Portuguese names are now in the game 👍
